net/lnet/lnet_xo.c

```diff
@@ -637,13 +636,11 @@ static const struct m0_net_xprt_ops nlx_xo_xprt_ops = {
    @{
  */
 
-#ifdef ENABLE_LUSTRE
```

Comment:
If this is removed, I am not sure if this will compile when there is no Lustre installed.

Reply:
@papan-singh : Can you verify it?

Reply:
Currently, Lustre is a dependent component.
But in future, we may remove that dependency.

Let's keep it in mind that Motr should work with Lustre in future.

Reply:
Verified 
-bash-4.2$ rpm -qa | grep lustre
lustre-client-devel-2.12.4-99.el7.x86_64
lustre-client-2.12.4-99.el7.x86_64
kmod-lustre-client-2.12.4-99.el7.x86_64
-bash-4.2$ sudo rpm -e lustre-client-devel-2.12.4-99.el7.x86_64
-bash-4.2$ sudo rpm -e lustre-client-2.12.4-99.el7.x86_64
-bash-4.2$ sudo rpm -e kmod-lustre-client-2.12.4-99.el7.x86_64
-bash-4.2$ ./autogen.sh
-bash-4.2$ ./configure --without-lustre
-bash-4.2$ make